lib/rails_param/param.rb

```diff
@@ -83,9 +83,13 @@ def coerce(param, type, options = {})
         return Integer(param) if type == Integer
         return Float(param) if type == Float
         return String(param) if type == String
-        return Date.parse(param) if type == Date
-        return Time.parse(param) if type == Time
-        return DateTime.parse(param) if type == DateTime
+        if [Date, DateTime, Time].include? type
```

Comment:
This is generating the `[Date, DateTime, Time]` array every time we check a parameter.
Please move this into a frozen constant and use that one instead.